pkg/permission/permission.go

```diff
@@ -27,6 +27,8 @@ const (
 	ListAllSpaces string = "list-all-spaces"
 	// CreateSpace is the hardcoded name for the create space permission
 	CreateSpace string = "create-space"
+	// CreatePublicLink is the hardcoded name for the publiclink.create permission
+	CreatePublicLink string = "publiclink.create"
```

Comment:
We should decide on on how we want to name permission. Last new implemented permissions go for `Drive.ReadWriteEnabled` (uppercase) therefore we should make this uppercase too, or change existing permissions to go lowercase.